build-system/compile/bundles.config.extensions.json

```diff
@@ -89,11 +89,10 @@
   },
```

Comment:
Confirming that the following are intentionally unconsolidated?
- `amp-youtube` 
- `amp-video`
- `amp-inline-gallery`

Reply:
`amp-inline-gallery` is there. But great catch: I still need `amp-youtube` and `amp-video`!

Reply:
Well, actually no. Sorry for confusion. I wanted to hold off `amp-video` and `amp-youtube` for now since they don't really have layout shift problem and the default `display` for the underlying components `display:inline`. What do you think?

Reply:
Added video and youtube stylesheets.